torch/optim/lr_scheduler.py

```diff
@@ -165,38 +165,55 @@ def get_lr(self):
 
 class CosineAnnealingLR(_LRScheduler):
     r"""Set the learning rate of each parameter group using a cosine annealing
-    schedule, where :math:`\eta_{max}` is set to the initial lr and
+    schedule, where :math:`\eta_{max}` is set to the initial lr,
+    :math:`T_{mult}` is the multiplicative factor of T_max and
     :math:`T_{cur}` is the number of epochs since the last restart in SGDR:
 
     .. math::
 
         \eta_t = \eta_{min} + \frac{1}{2}(\eta_{max} - \eta_{min})(1 +
-        \cos(\frac{T_{cur}}{T_{max}}\pi))
+        \cos(\frac{T_{cur}}{T_{max}\times T_{mult}}\pi))
```

Comment:
Can you omit the `\times` or use `\cdot` instead? That's likely to render clearer in the docs